### PR TITLE
Issue 92

### PR DESCRIPTION
**Pull request by LightAllWorld**
This PR fixes issue #92 :

It adds an setter `set_tesseract_path` about tesseract path.

### CODE REVIEW

File: deepdoctection/utils/file_utils.py [hunk 1]
```diff
@@ -249,8 +249,23 @@ def get_detectron2_requirement() -> Requirement:
 
 
 # Tesseract related dependencies
-
 _TESS_AVAILABLE = which("tesseract") is not None
+_TESS_PATH = "tesseract"
+
+def set_tesseract_path(tesseract_path: str="tesseract"):
+    global _TESS_AVAILABLE
```
Comment (JaMe76) [inline deepdoctection/utils/file_utils.py:256]: Please add a short docstring, e.g. what path to add for installation with Anaconda

File: deepdoctection/utils/file_utils.py [hunk 2]
```diff
@@ -249,8 +249,23 @@ def get_detectron2_requirement() -> Requirement:
 
 
 # Tesseract related dependencies
-
 _TESS_AVAILABLE = which("tesseract") is not None
+_TESS_PATH = "tesseract"
+
+def set_tesseract_path(tesseract_path: str="tesseract"):
```
Comment (JaMe76) [inline deepdoctection/utils/file_utils.py:255]: function will need a typing return statement:

```
set_tesseract_path(tesseract_path: str="tesseract") -> None:
```

**Review by JaMe76 — CHANGES_REQUESTED** [review]
Thanks for your PR!

1.) To make the PATH changes to tesseract effective in predictions _TESS_PATH needs to be imported to `extern.tessocr` and _TESS_PATH must replace "tesseract" in line 96. 

```
cmd_args += ("tesseract", input_file_name, output_file_name_base, "-l", lang)
```

2.) Can you add `set_tesseract_path` to `deepdoctection.__init__.py`  in _IMPORT_STRUCTURE dict in line 265? All modules are imported in a lazy manner, so not adding this function here would result in import error when invoked.   

3.) I've added some small remarks in your changes. 

Once merged, I would convert the original issue into the discussion section so users from Anaconda installs do not need to dig into closed issues. 

Thanks

**Comment by LightAllWorld** [comment]
Thanks for your comments.

1. Added doc strings about `set_tesserect_path` and `_TESS_PATH'.

2. Replaced extern.tessocr's "tesserect" in line 96.

3. Add `set_tesseract_path' to `deepdoctection.__init__.py` in 265 line.

**Review by JaMe76 — APPROVED** [review]
Thanks for your updates. 

Will be merged.

**Pull request merged**